src/org/usfirst/frc/team2706/robot/Robot.java

```diff
@@ -54,9 +58,14 @@ public void robotInit() {
         // Instantiate the robot subsystems
         driveTrain = new DriveTrain();
         // camera = new Camera(Camera.CAMERA_IP);
+        
+      //New bling system class.
```

Comment:
// New bling system class, make sure to have a space after the //, also its shifted a little bit back too far